packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -568,10 +537,9 @@ If the session is invalid, a `404 Not Found` response will be returned.
 ```
 
 #### Option: Use the HTTP Header Prefer
-Okta now supports [the HTTP Header `Prefer`](https://tools.ietf.org/html/rfc7240#section-4.2) in [the Sessions API for refreshing sessions](/docs/reference/api/sessions/#refresh-current-session). You can extend the session lifetime, but skip any processing work related to building the response body.
-
-##### Request Example
+Okta now supports [the HTTP Header `Prefer`](https://tools.ietf.org/html/rfc7240#section-4.2) in [the Sessions API for refreshing sessions](/docs/reference/api/sessions/#refresh-current-session). You can extend the session lifetime, but skip any processing work that is related to building the response body.
```

Comment:
Okta now supports [the HTTP Header `Prefer`](https://tools.ietf.org/html/rfc7240#section-4.2) in the [Sessions API for refreshing sessions](/docs/reference/api/sessions/#refresh-current-session).